src/pages/index.hbs

```diff
@@ -109,44 +109,73 @@
 			<div class="bg-blue-200 shadow-inner h-full w-full absolute top-0" style="transform: skewY(9deg); z-index: -1">
 			</div>
-			<div class="max-w-6xl px-4 md:px-12 lg:px-24 mx-auto">
-				<h2 id="betaphase" class="text-3xl md:text-3xl lg:text-4xl mb-10 -mt-8 sm:-mt-16 md:-mt-20 lg:-mt-24">
-					Kostenlose Beta-Phase
+			<div class="mx-auto max-w-3xl relative md:text-center">
+				<h2 class="text-4xl md:text-5xl mb-6 md:mb-12 px-4">
+					Preise
 				</h2>
-				<p class="mb-6 text-lg md:text-xl">
-					Melde dich jetzt zur Beta-Phase der Urlaubsverwaltung an und mache dir
-					selbst ein Bild.
+				<p class="text-2xl mb-12 px-4 text-gray-900 font-light">
+					Preisgestaltung so einfach wie die Urlaubsverwaltung.<br class="hidden sm:block md:hidden" />
+					Ein Paket, ein Preis.
+				</p>
+				<div class="border border-blue-100 py-8 px-4 mb-12 mx-4 text-2xl text-gray-900 bg-blue-100 bg-opacity-50 font-light">
+					<p class="mb-4 flex items-center md:justify-center">
+						<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" class="w-6 h-6 fill-current mr-2">
+							<path style="fill-rule: evenodd;"
+								d="M16.8792928,9.09695343 C18.6654343,9.4976045 20,11.09295 20,13 C20,15.209139 18.209139,17 16,17 L5,17 C2.23857625,17 0,14.7614237 0,12 C0,9.58046798 1.71857515,7.56233069 4.00162508,7.09968852 C4.00054449,7.06659179 4,7.03335948 4,7 C4,5.34314575 5.34314575,4 7,4 C7.55384606,4 8.07263826,4.1500834 8.51792503,4.41179863 C9.4182103,3.53797709 10.6462795,3 12,3 C14.7614237,3 17,5.23857625 17,8 C17,8.37684164 16.9583108,8.74394625 16.8792928,9.09695343 Z">
+							</path>
+						</svg>
+						&nbsp;<span class="">30 Tage testen</span>
+					</p>
+					<p class="mb-6">
+						Melde dich kostenlos und völlig unverbindlich an.<br />
+					</p>
+					<p class="mb-8 md:mb-12">
+						<a href="https://registry.beta.urlaubsverwaltung.cloud"
+							class="inline-block bg-blue-600 text-blue-100 px-4 py-2 font-light rounded-md text-lg">
+							Jetzt kostenlos testen
+						</a>
+					</p>
+					<div class="mx-auto text-left transform md:translate-x-1/4">
+						<ul>
+							<li class="flex items-center">
+								<svg class="w-6 h-6 mr-2" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 13l4 4L19 7"></path></svg>
+								Voller Funktionsumfang
+							</li>
+							<li class="flex items-center">
+								<svg class="w-6 h-6 mr-2" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 13l4 4L19 7"></path></svg>
+								Daten DSGVO-konform abgesichert
+							</li>
+						</ul>
+					</div>
+				</div>
+				<p class="px-4 text-2xl text-gray-900 font-light">
+					Nach der kostenlosen Testphase
```

Comment:
Ich glaube, man kann den Satz falsch verstehen: So nach dem Motto es geht automatisch über in das Bezahlmodell.